pdfpc/pdfpc-doc.tex

```diff
@@ -95,55 +101,48 @@ \section*{License}
     \item ``pdfpc-doc.pdf'' \emph{(compiled)}
 \end{itemize}
 
+
 \clearpage
 
 \part{The documentation}
 \section*{Loading \sty{pdfpc}}
 
-Load \sty{pdfpc} by adding \cmd{\usepackage[\param{options}]\{pdfpc\}} to your
-preamble.
+Load \sty{pdfpc} by adding \cmd{\usepackage[\param{options}]\{pdfpc\}} to your preamble.
 
 The following options may be given as comma-separated list:
 
 \begin{itemize}
```

Comment:
I think it would be nice to replace this list with a two-column table listing the pdfpc.sty options alongside their pdfpc command-line equivalents (probably only long-name variants).

Reply:
Updated, but I'm not sure how I feel about the way I've formatted this. How does it look to you?